<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortDir</class>
 <widget class="QWidget" name="sortDir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icos.qrc">
    <normaloff>:/new/prefix1/ICO/fileSortIco.ico</normaloff>:/new/prefix1/ICO/fileSortIco.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>文件夹路径：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameFormatEdit"/>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QPushButton" name="startSort">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>开始整理</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>或</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="fileDir">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>格式：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fileDirEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="endAddEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>指定文件后缀：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="choosePic">
     <property name="text">
      <string>图片</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="chooseTxt">
     <property name="text">
      <string>txt</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;文件名格式用 - 隔开。双 -，即 -- 将保留一个 -，否则时间之间没有-&lt;/p&gt;&lt;p&gt;yy表示年&lt;/p&gt;&lt;p&gt;mm表示月&lt;/p&gt;&lt;p&gt;dd表示日&lt;/p&gt;&lt;p&gt;hh表示当前时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icos.qrc"/>
 </resources>
 <connections/>
</ui>
